runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilitySession.java

```diff
@@ -49,7 +49,12 @@ void incrementLapCount(Client client)
 	{
 		lastLapCompleted = Instant.now();
 		++totalLaps;
+		refreshLapsTillLvl(client);
 
+	}
+
+	void refreshLapsTillLvl(Client client)
```

Comment:
`calculateLapsToLevel` or `reloadLapsToLevel` would be a better name.

Reply:
renamed in latest commit